test/lib/file.test.js

```diff
@@ -154,8 +154,17 @@ describe('File', function () {
     it('is a getter that returns a string', function () {
       assume(file.location).to.be.a('string');
     });
+    it('returns an external location', function () {
```

Comment:
Nvm, I see https://github.com/bigpipe/bigpipe/commit/88b9c129e0e43ac863187e7442f75a1083a3565a#diff-17dc9a452db5488b5cc544f82988d51bR32 this would make sense as removal of the `extname` is required but also breaks the current implementation